managers/muse.py

```diff
@@ -68,22 +69,25 @@ def identifyReadableVersions(self):
 
     def addReadableLinks(self):
         if self.pdfDownloadURL:
-            self.record.add_has_part_link(
+            add_has_part_link(
```

Comment:
hmm - this is interesting - almost seems like a function that should be a part of the record class but we can pick that up later!